test/test-template-engine.js

```diff
@@ -761,6 +723,86 @@ describe('svg updaters', () => {
 });
 
 describe('rendering errors', () => {
+  describe('templating', () => {
+    it.skip('throws when attempting to interpolate within a style tag', () => {});
```

Comment:
Added a couple placeholders — we should make this even more strict. For a future PR though.